### PR TITLE
Add semantic type icon/name to dimension info popovers

### PR DESCRIPTION
**Pull request by daltojohnso**
This PR replaces the `DimensionLabel` found in the `DimensionInfoPopover` with `DimensionSemanticTypeLabel`. Previously, `DimensionLabel` showed the name of the dimension; now, `DimensionSemanticTypeLabel` shows the semantic type of a dimension, if one exists.

Screen Shot 2022-01-17 at 4 48 17 PM
Screen Shot 2022-01-17 at 4 48 09 PM

**Testing**
1. Navigate to the Products table
2. Hover over headers of the table to view the dimension info popover -- you should now see a label for the associated field's semantic type
3. Go to admin --> data model --> products table and update the semantic type of some fields.
4. Return to the Products table to see the updated labels in the dimension info popover

### CODE REVIEW

**Comment by mazameli** [comment]
Looks good to me! Thanks for fixing this.

File: frontend/src/metabase/icon_paths.ts [hunk 1]
```diff
@@ -90,6 +90,13 @@ export const ICON_PATHS: Record<string, any> = {
...
+    attrs: {
+      fillRule: "evenodd",
+    },
+  },
```
Comment (nemanjaglumac) [inline frontend/src/metabase/icon_paths.ts:99]: 🎈🎈🎈

**Review by gusaiani — APPROVED** [review]
Looks great 👍

**Review by nemanjaglumac — APPROVED** [review]
Thanks for adding unit tests and for updating Cypress test.

**Pull request merged**